exit.go

```diff
@@ -135,6 +154,53 @@ func (te *TunaExit) handleSession(session *smux.Session, conn net.Conn) {
 				}
 			}
 		}()
+	} else {
+		go func() {
+			var np *nknsdk.NanoPay
+			for {
+				time.Sleep(time.Second * 1)
```

Comment:
1. Same as above, remove duplicate code
2. There are logs of debug print and modified constants, do not commit before removing/recovering such code